File: Datasets/HumanEval/HumanEval_Java/22_P13.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;


class FilterIntegers {
    /**
     * Filter given list of any Java values only for integers
     * >>> filter_integers(['a', 3.14, 5])
     * [5]
     * >>> filter_integers([1, 2, 3, 'abc', {}, []])
     * [1, 2, 3]
     * @param values a list any Java values
     * @return the list fitered for only integers
     *
     */
    public static List<Object> filterIntegers(List<Object> values) {

File: Cleared_Dataset/HumanEval_Py/22_P13.py
from typing import List, Any


def filter_integers(values: List[Any]) -> List[int]:
    """ Filter given list of any python values only for integers.

    args:
        values: list of any python values
    
    returns:
        list of integers from input list `values'

    examples:
    >>> filter_integers(['a', 3.14, 5])
    [5]
    >>> filter_integers([1, 2, 3, 'abc', {}, []])
    [1, 2, 3]
    """
